src/css/cs/items/_kitchensink.scss

```diff
@@ -347,6 +347,10 @@
     }
   }
 
+  .uc-white-space-pre-wrap {
+    white-space: pre-wrap;
```

Comment:
Interesting read at https://developer.mozilla.org/en-US/docs/Web/CSS/white-space#Values on the differences. You might want to test with 1000's of spaces.